generatedocs.py

```diff
@@ -81,10 +116,32 @@ def get_lines_and_content_from_file(asset_file, regex, look_for_header = False):
 
       # If we are in header comment, split into header and description
       if look_for_header and is_header_comment:
-        comment = line.split(LUA_COMMENT)[1]
+        # Remove the beginning of the line that marks it being a Lua comment
```

Comment:
You can use the built in `lstrip` to remove leading whitespaces
http://docs.python.org/library/stdtypes.html#str.lstrip